### PR TITLE
// BO : UIkit integration & some JS fixes

### PR DESCRIPTION
**Pull request by kelu95**

### CODE REVIEW

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -874,7 +873,7 @@ var specificPrices = (function() {
 						<td>'+ specific_price.impact +'</td>\
 						<td>'+ specific_price.period +'</td>\
 						<td>'+ specific_price.from_quantity +'</td>\
```
Comment (djfm) [inline admin-dev/themes/default/js/bundle/product/form.js]: Multiline strings in JS are [not safe](https://google.github.io/styleguide/javascriptguide.xml#Multiline_string_literals), you could use transpiled ES6 and [template strings](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/template_strings) instead.

File: admin-dev/themes/new-theme/js/theme.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import '../node_modules/magnific-popup/dist/magnific-popup.css';
 import '../node_modules/PrestaKit/dist/css/bootstrap-prestashop-ui-kit.css';
 import '../node_modules/PrestaKit/dist/css/jquery.growl.css';
```
Comment (djfm) [inline admin-dev/themes/new-theme/js/theme.js]: Why not

```javascript
import 'PrestaKit/dist/css/jquery.growl.css';
```

etc.

File: admin-dev/themes/default/js/bundle/product/form.js (hunk 1 above)
Comment (kelu95) [inline admin-dev/themes/default/js/bundle/product/form.js]: done

File: admin-dev/themes/new-theme/js/theme.js (hunk 2 above)
Comment (Shudrum) [inline admin-dev/themes/new-theme/js/theme.js]: Because I just don't know it was possible ^^ TY!

**Pull request merged**